EngineLayer/NonSpecificEnzymeSearch/NonSpecificEnzymeSequencesToActualPeptides.cs

```diff
@@ -71,7 +71,7 @@ protected override MetaMorpheusEngineResults RunSpecific()
                             }
                         }
 
-            var HACKYYYYYYDigestionParams = CollectionOfDigestionParams.First();
+            var HACKYYYYYYDigestionParams = collectionOfDigestionParams.First();
```

Comment:
you might want to remove this!

Reply:
I think we should hold off on removing this until we change the way that precursor masses are designated. I might take a crack at that tonight if I have time.

Reply:
Up to you! The thing is that if it's still there, the non-specific searches would not work for file-specific parameters..